packages/storybook/src/stories/ino-table/ino-table.stories.ts

```diff
@@ -23,113 +24,123 @@ export default {
-        <ino-table-header-cell column-id="box-office" label="Box Office"></ino-table-header-cell>
-        <ino-table-header-cell column-id="rating" label="Rating" not-sortable></ino-table-header-cell>
-        <ino-table-header-cell column-id="another1" label="Another Column" not-sortable></ino-table-header-cell>
-        <ino-table-header-cell column-id="another2" label="Another Column" not-sortable></ino-table-header-cell>
-      </tr>
-      <tr>
-        <td>1</td>
-        <td>The Bourne Identity</td>
-        <td>14.06.2002</td>
-        <td>$214 million</td>
-        <td>93%</td>
-        <td>Some information</td>
-        <td>Some information</td>
-      </tr>
-      <tr>
-        <td>2</td>
-        <td>Sully</td>
-        <td>09.09.2016</td>
-        <td>$240.8 million</td>
-        <td>84%</td>
-        <td>Some information</td>
-        <td>Some information</td>
-      </tr>
-      <tr>
-        <td>3</td>
-        <td>The Martian</td>
-        <td>02.10.2015</td>
-        <td>$630.2 million</td>
-        <td>94%</td>
-        <td>Some information</td>
-        <td>Some information</td>
-      </tr>
-      <tr>
-        <td>4</td>
-        <td>John Wick: Chapter 3</td>
-        <td>17.05.2019</td>
-        <td>$326.7 million</td>
-        <td>92%</td>
-        <td>Some information</td>
-        <td>Some information</td>
-      </tr>
-      <tr>
-        <td>5</td>
-        <td>10 Cloverfield Lane</td>
-        <td>11.03.2016</td>
-        <td>$110.2 million</td>
-        <td>87%</td>
-        <td>Some information</td>
-        <td>Some information</td>
-      </tr>
-      <tr>
-        <td>6</td>
-        <td>Thor: Ragnarok</td>
-        <td>10.10.2017</td>
-        <td>$854 million</td>
-        <td>92%</td>
-        <td>Some information</td>
-        <td>Some information</td>
-      </tr>
-      <tr>
-        <td>7</td>
-        <td>The latest movie</td>
-        <td>01.01.2021</td>
-        <td>$900 million</td>
-        <td>95%</td>
-        <td>Some information</td>
-        <td>Some information</td>
-      </tr>
-    `)}
+  ${args.loading? html`<ino-progress-bar slot="loading-indicator" indeterminate debounce="200" active></ino-progress-bar>` : html``}
+  ${tableContent}
   </ino-table>
-`;
-
-Playground.args = {
-  loading: false,
-  sortColumnId: 'id',
-  noHover: false,
-  stickyHeader: false
-}
-Playground.argTypes = {
-  sortColumnId: {
-    options: ['id', 'name', 'release', 'box-office', 'rating'],
-    control: {
-      type: 'select'
-    }
-  }
-}
-withSortDirection(Playground, 'sortDirection', 'asc')
-
+`);
 
-export const SelectionWithCheckboxes = () => {
+export const Playground = template.generatePlaygroundStory();
+export const Loading = template.generateStoryForProp('loading', true);
+export const NoHover = template.generateStoryForProp('noHover', true);
+export const SortColumnId = template.generateStoryForProp('sortColumnId', 'id');
```

Comment:
I think we should add an extra text information that the sorting function is not part of the `ino-table` and have to provided by consumer.

Reply:
I put the disclaimer in the description of the merged story `SortColumnIdAndSortDirection`